corehq/feature_previews.py

```diff
@@ -106,3 +117,13 @@ def enable_callcenter(domain_name, checked):
         '<a href="https://help.commcarehq.org/display/commcarepublic/Custom+User+Data">custom user data</a>.'
     ),
 )
+
+
+VELLUM_ADVANCED_ITEMSETS = FeaturePreview(
+    slug='advanced_itemsets',
+    label=_("Custom Single and Multiple Answer Questions"),
+    description=_(
+        "Allows display of case sharing groups or locations as choices in Single Answer or"
+        " Multiple Answer questions."
```

Comment:
I don't think this will display case sharing groups or locations in the dropdown like lookup tables are displayed. It just allows you to more easily write some custom xpaths for doing that